components/project-card/project-card-detailed.jsx

```diff
@@ -0,0 +1,197 @@
+      "mt-3": this.props.thumbnail
+    });
+
+    return <a href={this.props.contentUrl} target="_blank" className={classnames} onClick={() => this.handleVisitBtnClick()}>Visit</a>;
+  }
+
+  renderTimePosted() {
+    if (!this.props.created && !this.props.publishedBy) return null;
+
+    let timePosted = this.props.created ? ` ${moment(this.props.created).format(`MMM DD, YYYY`)}` : null;
+    let publishedBy = this.props.publishedBy ? <span> by {this.props.publishedBy}</span> : null;
+
+    return (
+      <p><small className="time-posted d-block">
+        Added{timePosted}{publishedBy}
+      </small></p>
+    );
+  }
+
+  renderActionPanel() {
+    let twitterUrl = `https://twitter.com/intent/tweet?text=${encodeURIComponent(this.props.title)}&url=${encodeURIComponent(window.location.href)}`;
+
+    return (
+      <div className="action-panel pb-3 mb-3">
+        <div className="d-flex share">
+          <BookmarkControl id={this.props.id}
+                           title={this.props.title}
+                           isBookmarked={this.props.isBookmarked}
+                           updateCardBookmarkedState={(bookmarked) => { this.updateCardBookmarkedState(bookmarked); }} />
+          <a href={twitterUrl} onClick={evt => this.handleTwitterShareClick(evt) } className="btn twitter-share d-inline-block align-self-center mx-3"></a>
+        </div>
+      </div>
+    );
+  }
+
+  renderTopHeader() {
+    return <div className="col-12 mb-3">
+            <div className="row">
+              <div className="col-12 col-sm-8">
+                <Title title={this.props.title} />
+                <Creators creators={this.props.creators} showLabelText={true} />
+              </div>
+            </div>
+          </div>;
+  }
+
+  renderLeftColumn() {
+    let wrapperClassnames = classNames({
+      "col-12": true,
+      "col-md-8": true
+    });
+
+    return <div className={wrapperClassnames}>
+              <Thumbnail thumbnail={this.props.thumbnail} />
+              { this.renderVisitButton() }
+              <Description description={this.props.description} className="mt-3" />
+              <WhyInteresting interest={this.props.interest} />
+              { this.renderTimePosted() }
+              <IssuesAndTags issues={this.props.issues} tags={this.props.tags} />
+            </div>;
+  }
+
+  renderRightColumn() {
+    let wrapperClassnames = classNames({
+      "col-12": true,
+      "col-md-4": true
+    });
+
+    return <div className={wrapperClassnames}>
+              { this.renderActionPanel() }
+              <GetInvolved getInvolved={this.props.getInvolved}
+                           getInvolvedUrl={this.props.getInvolvedUrl}
+                           helpTypes={this.props.helpTypes}
+                           sendGaEvent={(category, action, transport) => this.sendGaEvent(category, action, transport)} />
+            </div>;
+  }
+
+  render() {
+    let wrapperClassnames = classNames({
```

Comment:
Why not do what you already do here? https://github.com/mozilla/network-pulse/pull/617/files#diff-9ce64733fe7a204e270fad8337eecef5R76 (pass most of them in as string data)